tests/storage/datavolume.go

```diff
@@ -868,6 +868,19 @@ var _ = Describe(SIG("DataVolume Integration", func() {
 						Name:      snap.Name,
 					},
 				}
+
+				By("Waiting for snapshot to have restore size")
+				Eventually(func() bool {
+					snap, err = virtClient.KubernetesSnapshotClient().
+						SnapshotV1().
+						VolumeSnapshots(snap.Namespace).
+						Get(context.Background(), snap.Name, metav1.GetOptions{})
+					Expect(err).ToNot(HaveOccurred())
+					return snap.Status != nil && snap.Status.RestoreSize != nil
+				}, 90*time.Second, 1*time.Second).Should(BeTrue())
```

Comment:
could achieve this in a cleaner way https://github.com/kubevirt/kubevirt/blob/c69f52dd822313c05698f9a1408df421dda7a481/tests/framework/matcher/instancetype.go#L15-L21

Reply:
that way you would also not fail over the Expect call that's inside the eventually
(if under certain weird conditions it decided to fail)

Reply:
ack